### PR TITLE
Creating an HPC Cluster - PC2.9

### PR DESCRIPTION
**Pull request by trnh**
*Issue #, if available:*

*Description of changes:*
- Rearrange sections so better tutorial flows
- Add new documentation links for more information
- Redo new PC lab and output to reflect PC2.9 change
- Remove Behind the scene section ASG to reflect new change in PC2.9
- Add How to update PC section to reflect a new change in PC2.8 that wasn't there last year.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by perifaws — DISMISSED** [review]

**Review by perifaws — DISMISSED** [review]

File: content/03-hpc-aws-parallelcluster-workshop/04-configure-pc.md [hunk 1]
```diff
@@ -28,36 +29,44 @@ IFACE=$(curl --silent http://169.254.169.254/latest/meta-data/network/interfaces
...
-aws_region_name = ${REGION}
+aws_region_name = us-east-1
 
 [global]
 cluster_template = default
 update_check = false
 sanity_check = true
 
+[vpc public]
+vpc_id = vpc-aff37ad7
```
Comment (sean-smith) [inline content/03-hpc-aws-parallelcluster-workshop/04-configure-pc.md:48]: This shouldn't be here

File: content/03-hpc-aws-parallelcluster-workshop/04-configure-pc.md [hunk 2]
```diff
@@ -28,36 +29,44 @@ IFACE=$(curl --silent http://169.254.169.254/latest/meta-data/network/interfaces
...
+```
+```bash
 cd ~/environment
+```
+
+```bash
 cat > my-cluster-config.ini << EOF
 [aws]
-aws_region_name = ${REGION}
+aws_region_name = us-east-1
```
Comment (sean-smith) [inline content/03-hpc-aws-parallelcluster-workshop/04-configure-pc.md:40]: same here

**Review by sean-smith — CHANGES_REQUESTED** [review]

File: content/03-hpc-aws-parallelcluster-workshop/04-configure-pc.md (hunk 1 above)
Comment (perifaws) [reply content/03-hpc-aws-parallelcluster-workshop/04-configure-pc.md:48]: I overlooked that one

File: content/03-hpc-aws-parallelcluster-workshop/04-configure-pc.md (hunk 2 above)
Comment (perifaws) [reply content/03-hpc-aws-parallelcluster-workshop/04-configure-pc.md:40]: +1

**Review by sean-smith — APPROVED** [review]

**Pull request merged**